modules/monitoring-understanding-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_content-type: CONCEPT
+[id="understanding-alert-routing-for-user-defined-projects_{context}"]
+= Understanding alert routing for user-defined projects
+
+[role="_abstract"]
+As a cluster administrator, you can enable alert routing for user-defined projects. After doing so, you can allow users to configure alert routing for their user-defined projects. Users can then create and configure user-defined alert routing by editing the `AlertmanagerConfig` CRD.
```

Comment:
Technically you're not "editing the AlertmanagerConfig CRD" but rather "editing AlertmanagerConfig objects".

```suggestion
As a cluster administrator, you can enable alert routing for user-defined projects. After doing so, you can allow users to configure alert routing for their user-defined projects. Users can then create and configure user-defined alert routing by creating/editing `AlertmanagerConfig` objects.
```

Reply:
Fixed